content/en/internal/configuration-database/_index.md

```diff
@@ -1,44 +1,26 @@
 ---
 title: "Configuration Database"
-description: "Configuration Database (GOCDB) documentation"
+description: "Configuration Database (GOCDB)"
 weight: 30
 type: "docs"
 ---
 
-{{% pageinfo %}} This documentation about the Configuration Database is a work
-in progress, more documentation is currently available at
-[EGI's wiki](https://wiki.egi.eu/wiki/GOCDB). {{% /pageinfo %}}
+## What is it?
 
-The EGI Configuration Database (previously known as the GOCDB) contains general
-information about the sites participating to the production infrastructure.
+The [EGI Configuration Database](https://goc.egi.eu) (previously known as
+GOCDB) contains general information about the sites participating in the EGI
+Cloud infrastructure.
 
-It's a central registry to record information about the topology of the EGI
-e-Infrastructure. This includes entities such as Operations Centres, Resource
-Centres, service endpoints and their downtimes, contact information and roles of
-users responsible for operations at different levels.
+It is a central registry to record information about the topology of the EGI
+Cloud infrastructure. This includes entities such as Operations Centres, Resource
+Centres, service endpoints together with their downtimes, contact information,
+and user roles responsible for operations at various levels.
 
-It's used by all the actors (end-users, site managers, NGI mangers, support
-teams, VO managers), by other tools and by third party middleware to discover
-information about the infrastructure topology.
+The configuration database is used by all the actors (end-users, site managers,
+NGI mangers, support teams, VO managers), by other tools, and by third party
+middleware to discover information about the infrastructure topology.
 
-## Identity card
```

Comment:
Do not remove the id card. The wiki is to be deprecated and docs should be moved here instead!

Reply:
@thebe14 please fix this one to be able to merge

Reply:
okay. I just looked at the linked wiki page and all the info is there. added it back. also added a todo to all the other services to add a similar card.